README.md

```diff
@@ -17,6 +17,9 @@ A templated Django messaging library
     python manage.py migrate ilmoitin
     ```
 
+Use version >=0.6.x for a project based on Django 3.x
+Use version 0.5.x for a project based on Django 2.x
```

Comment:
missing a line in between for them to be on separate lines 😬 
![image](https://user-images.githubusercontent.com/24206160/88791066-cbf51d80-d1a1-11ea-8588-fc8f9e53482d.png)